index.html

```diff
@@ -0,0 +1,445 @@
+            <article>
+              <p>This guide assumes you have the following basic background:</p>
+        
+              <ul>
+                <li>
+                  A general understanding of the Internet and the World Wide Web (WWW).
+                </li>
+                <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+                <li>
+                  Some programming experience. If you are new to programming, try one of
+                  the tutorials linked on the main page about JavaScript.
+                </li>
+              </ul>
+            </article>
+          </section>
+          <section class="main-section" id="JavaScript_and_Java">
+            <header>JavaScript and Java</header>
+            <article>
+              <p>
+                JavaScript and Java are similar in some ways but fundamentally different
+                in some others. The JavaScript language resembles Java but does not have
+                Java's static typing and strong type checking. JavaScript follows most
+                Java expression syntax, naming conventions and basic control-flow
+                constructs which was the reason why it was renamed from LiveScript to
+                JavaScript.
+              </p>
+        
+              <p>
+                In contrast to Java's compile-time system of classes built by
+                declarations, JavaScript supports a runtime system based on a small
+                number of data types representing numeric, Boolean, and string values.
+                JavaScript has a prototype-based object model instead of the more common
+                class-based object model. The prototype-based model provides dynamic
+                inheritance; that is, what is inherited can vary for individual objects.
+                JavaScript also supports functions without any special declarative
+                requirements. Functions can be properties of objects, executing as
+                loosely typed methods.
+              </p>
+              <p>
+                JavaScript is a very free-form language compared to Java. You do not
+                have to declare all variables, classes, and methods. You do not have to
+                be concerned with whether methods are public, private, or protected, and
+                you do not have to implement interfaces. Variables, parameters, and
+                function return types are not explicitly typed.
+              </p>
+            </article>
+          </section>
+          <section class="main-section" id="Hello_world">
+            <header>Hello world</header>
+            <article>
+              To get started with writing JavaScript, open the Scratchpad and write your
+              first "Hello world" JavaScript code:
+              <code
+                >function greetMe(yourName) { alert("Hello " + yourName); }
+                greetMe("World");
```

Comment:
Please consider closing the `code` tag for your code to be in compliance with HTML best practices. e.g
`<code>function greetMe(yourName) { alert("Hello " + yourName); } greetMe("World"); </code>`